packages/api/tests/models/test-collections-model.js

```diff
@@ -47,24 +37,20 @@ test('Collection.exists() returns true when a record exists', async (t) => {
   const name = randomString();
   const version = randomString();
 
-  await manager.create(fakeCollectionFactory({ name, version }));
-
-  const collectionsModel = new Collection();
+  await collectionsModel.create(fakeCollectionFactory({ name, version }));
 
   t.true(await collectionsModel.exists(name, version));
 });
 
 test('Collection.exists() returns false when a record does not exist', async (t) => {
-  const collectionsModel = new Collection();
-
-  t.false(await collectionsModel.exists(randomString()));
+  t.false(await collectionsModel.exists(randomString(), randomString()));
```

Comment:
Interesting. The change here is definitely valid based on the signature of `collection.exists`, but how was this test passing before? I guess because it was treating version as undefined and trying to look  a collection where name is `name` and version is `undefined`. 

Regardless, great catch!

Reply:
Right, we could probably just as easily use `collectionModel.exists()` (no arguments) and have the test pass because a model with both name and version equal to undefined would not exist either.